styles.css

```diff
@@ -0,0 +1,824 @@
+  font-weight: bolder;
+  text-align: center;
+}
+
+.about-courses {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  max-width: 100%;
+}
+
+.about-courses h2 {
+  font-weight: bolder;
+  text-align: center;
+}
+
+.about-img-courses {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  width: 100vw;
+  height: 20rem;
+  align-items: center;
+  top: 0;
+}
+
+.about-img2,
+.about-img1 {
+  position: relative;
+  background: #000 url(img/system4.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 0 4rem;
+  color: #fff;
+  height: 17rem;
+  width: 30rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.5rem;
+  text-align: center;
+  max-width: 100%;
+}
+
+.about-img2 > div:hover {
+  display: none;
+}
+
+.about-img1 > div:hover {
+  display: none;
+}
+
+.color-overlay {
+  width: 100%;
+  height: 100%;
+  background: red;
+  opacity: 0.5;
+  position: absolute;
+}
+
+.pro p {
+  max-width: 100%;
+  margin: 0 0 5rem 0;
+  padding: 1rem;
+  font-size: 0.8rem;
+}
+
+.about-img-courses li p {
+  z-index: 1;
+  font-weight: bold;
+}
+
+.para {
+  position: relative;
+  border: 2px solid white;
+  max-width: 45rem;
+  line-height: 1.7;
+  margin-top: 1rem;
+  padding: 2rem;
+  background-color: #c4c4c4;
+  font-weight: 600;
+}
+
+.showcase h2 {
+  position: relative;
+  font-weight: 800;
+  padding-top: 2rem;
+  letter-spacing: 0.2rem;
+}
+
+.para1 {
+  position: relative;
+  max-width: 100%;
+  font-weight: 600;
+  line-height: 1.7;
+  margin-top: 1rem;
+  padding: 2rem;
+  color: #e85242;
+  font-size: 2rem;
+  padding-left: 0;
+}
+
+.program {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  background-color: #262930;
+  color: #fff;
+  padding: 1rem 0 0 0;
+}
+
+.courses {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 5rem 0 5rem;
+  height: 100%;
+}
+
+.pro {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 0.1rem;
+  background-color: #3c3f45;
+  color: #fff;
+}
+
+.img-logo {
+  width: 3rem;
+  align-self: center;
+}
+
+hr {
+  background-color: rgb(250, 19, 19);
+  font-weight: bolder;
+  width: 3rem;
+}
+
+.pro:hover {
+  border: 1px solid #fff;
+}
+
+u {
+  padding: 1rem 0 5rem 0;
+}
+
+.footer1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #414246;
+  color: #fff;
+  padding: 2.5rem;
+  max-width: 100%;
+}
+
+.footer-logos {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer-logos img {
+  margin: 0 0.5rem;
+}
+
+.footer2-img img {
+  max-width: 30%;
+}
+
+.footer-about {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #2e2e2e;
+  padding: 3.5rem 0;
+  max-width: 100%;
+  color: #fff;
+}
+
+.footer2 {
```

Comment:
- [ ] As stated by the previous reviewer here we need to replicate the footer design as much as possible. Lets see how we can fix this 💪 
- The logo and the footer text are too far wide apart this means that that there needs to be some margin around them so that they can appear to be at the center/closer to each other or you can even decide to reduce the `footer2` selector width.
- The background color also seems not to have been addressed yet. Remember, colors are part of the hard requirement for this project. This will also mean that you also need to change the text color.
-